src/cli/src/bin.ts

```diff
@@ -20,7 +20,8 @@ const argv = yargs
   })
   .group(['help', 'verbose'], 'Global:')
   .commandDir(path.join(__dirname, 'commands'), {
-    extensions: ['ts', 'js']
+    extensions: ['ts', 'js'],
+    exclude: /\.d\.ts$/
```

Comment:
I did this differently in the server package, but I like your solution better. Any chance you could apply this there as well?

https://github.com/paularmstrong/build-tracker/blob/93e3a31708790f020e3f65c6d5964cc52cc30b00/src/server/src/index.ts#L22

Reply:
Sure, done!